docs/installation/6-ldap.md

```diff
@@ -143,17 +143,29 @@ AUTH_LDAP_CACHE_TIMEOUT = 3600
 
 `systemctl restart netbox` restarts the Netbox service, and initiates any changes made to `ldap_config.py`. If there are syntax errors present, the NetBox process will not spawn an instance, and errors should be logged to `/var/log/messages`.
 
-For troubleshooting LDAP user/group queries, add the following lines to the start of `ldap_config.py` after `import ldap`.
+For troubleshooting LDAP user/group queries, add or merge the following [logging](/configuration/optional-settings.md#logging) configuration to `configuration.py`:
 
 ```python
-import logging, logging.handlers
 logfile = "/opt/netbox/logs/django-ldap-debug.log"
```

Comment:
A small nitpick, but can we get rid of this variable to just set the filename directly in the dict? I think it is slightly less confusing as we don't really do this sort of thing anywhere else in configuration.py (not that users aren't free to do whatever they like that is).

Reply:
Removed